controller/userController.js

```diff
@@ -0,0 +1,152 @@
+const userService = require('../services/userService');
+const middleNick = require('../util/nickMaker');
+const middleJwt = require('../middleware/jwt');
+
+const signUp = async(req,res) => {
+    try{
+        if(!req.body.email || !req.body.password || !req.body.phone_number || !req.body.birthday){
+            return res.status(400).json({message : 'KEY_ERROR'})
+        }
```

Comment:
```suggestion
    try{
        if(!req.body.email || !req.body.password || !req.body.phone_number || !req.body.birthday){
        const error = new Error("KEY_ERROR")
        error.statusCode = 400
        throw error

      }
```

에러 리턴하는 대신, throw 구문 이용해 보시고, throw한 에러가 try-catch문 내에서 어떻게 동작하는지 확인해 보세요!